src/api-client.ts

```diff
@@ -1,3 +1,5 @@
+import { version } from "../package.json";
```

Comment:
In theory this should work. I'm about 30% skeptical. We will test it once this gets published.

Reply:
Actually less skeptical now. It looks like `tsup` just hoists the value out of the `package.json` file when it bundles it, so when the library is installed and executed, this just refers to a constant.

<img width="294" alt="image" src="https://github.com/user-attachments/assets/da830987-bc5f-4f34-b5a7-85f1dfd99348">